### PR TITLE
fix OptionalGetElement shape inference when input is a tensor

### PR DESCRIPTION
**Pull request by liqunfu**
### Description
There is a bug in shape inference when the input is a tensor. 

### Motivation and Context
fix the bug and add a test for this case.

### CODE REVIEW

File: onnx/test/shape_inference_test.py [hunk 1]
```diff
@@ -6997,6 +6997,25 @@ def test_optional_sequence_has_element(self) -> None:
...
         )  # type: ignore
 
+    def test_tensor_get_element(self) -> None:
+        tensor_type_proto = helper.make_tensor_type_proto(
+            elem_type=TensorProto.DOUBLE, shape=[2, 1, 4]
+        )
+        input_tensor_val_info = helper.make_value_info(
+            name="input", type_proto=tensor_type_proto
+        )
+        output_tensor_val_into = helper.make_value_info(
```
Comment (gramalingam) [inline onnx/test/shape_inference_test.py:7007]: nit: `val_into` should be `val_info` I think

**Review by gramalingam — APPROVED** [review]
LGTM, thanks ... just one minor typo in naming.

**Review by gramalingam — APPROVED** [review]

**Comment by justinchuby** [comment]
@jcwchen will this be part of onnx 1.13.1?

**Comment by jcwchen** [comment]
> @jcwchen will this be part of onnx 1.13.1?

Unfortunately no. All included PRs for ONNX 1.13.1 are listed here: https://github.com/onnx/onnx/milestone/9?closed=1. We have announced the verification for final release candidate this Tuesday for community verification and planned to release next Tuesday.

**Pull request merged**